.github/workflows/push_pr.yml

```diff
@@ -28,11 +28,9 @@ jobs:
   compile_and_test:
     strategy:
       matrix:
-          flink: [ 1.17-SNAPSHOT ]
-          jdk: [ '8, 11' ]
+          flink: [ 1.18-SNAPSHOT ]
```

Comment:
curious why we don't use the real 1.18 (instead of `1.18-SNAPSHOT` for pr

Reply:
I am not sure, I agree that makes sense though.